functions/go/enforce-gatekeeper/validate.go

```diff
@@ -175,7 +174,58 @@ func parseResults(results []*opatypes.Result) (*framework.Result, error) {
 			}
 		}
 
-		out.Items = append(out.Items, item)
+		items = append(items, item)
+	}
+	sortResultItems(items)
+
+	return &framework.Result{
+		Items: items,
+	}, nil
+}
+
+// TODO(mengqiy): upstream this to the SDK
```

Comment:
Are you keeping track of these systematically. There are multiple PRs with TODOs for enhancing the SDK.

Reply:
https://github.com/GoogleContainerTools/kpt/issues/2195